src/apps/relay/dtls_listener.c

```diff
@@ -966,8 +944,6 @@ void setup_dtls_callbacks(SSL_CTX *ctx) {
   SSL_CTX_set_verify(ctx, SSL_VERIFY_PEER | SSL_VERIFY_CLIENT_ONCE, dtls_verify_callback);
 #endif
 
-  SSL_CTX_set_read_ahead(ctx, 1);
```

Comment:
afaik this is fine but can you add to the PR description why is this removed / not needed for future reference?